lib/huginn_agent/patches/coverage.rb

```diff
@@ -0,0 +1,19 @@
+require 'simplecov'
+
+if !ENV['COVERAGE']
+  require 'coveralls'
+  module Coveralls
+    module Configuration
+      def self.root
+	File.expand_path(File.join(Dir.pwd, '../..'))
```

Comment:
Can we make these paths relative to the file using something like `File.expand_path(File.join(File.dirname(__FILE__), '../..'))`, or is the point that we don't know where this file will be?